src/main/java/net/masterthought/cucumber/generators/AbstractPage.java

```diff
@@ -27,6 +17,12 @@
 import org.apache.velocity.app.event.EventCartridge;
 import org.apache.velocity.runtime.resource.loader.ClasspathResourceLoader;
 
+import java.io.*;
```

Comment:
Lets separate all changes to another PR which will be merged first and then we can reduce number of changes only to the important once.
Also be consistent :)